### ISSUE TITLE
clk_wiz_0.vhd is missing?

### ISSUE THREAD

**Issue by zoonono** [HoFong667/SSM2603-Proj_v1#1]
Hello HoFong,
I tried to run the code under Vivado 2014.4. But I got the following errors.
[Project 1-486] Could not resolve non-primitive black box cell 'clk_wiz_0' instantiated as 'uclk_wiz' ["C:/SSM2603_Proj_v1/SSM2603_Proj_v1.srcs/sources_1/imports/SSM2603-Proj_v1/top_module_v2.vhd":221]
It seems that clk_wiz_0.vhd is missing?
Thanks!

**Comment by HoFong667** [comment]
Yes, you need to instantiate two clock through clocking wizard in IP catalog. Please see the readme.

anyway, I use Vivado 2022 to develop this.

**Comment by zoonono** [comment]
Hi HoFong, thank you very much for your quick response and great suggestions. I will try to add those two clocks (5MHz, 24MHz) through clocking wizard in IP catalog.
